search/src/java/cz/incad/Kramerius/oai/DrKrameriusV4Writer.java

```diff
@@ -166,6 +177,27 @@ private void generateRecord(Context ctx) throws XMLStreamException, IOException
 
         writer.add(eventFactory.createEndElement(RECORD_QNAME, null));
     }
+    
+    private String getDCPolicy(String pid) throws IOException {
+        InputStream datastream = fedora.getDataStream(pid, "DC");
+        Document document = getXmlFromInputStream(datastream);
+        String rights = document.getElementsByTagName("dc:rights").item(0).getTextContent();
```

Comment:
Může se stát, že DC nebude vůbec obsahovat rights....a pak to sletí n NullPointerException - to prosím ošetři (měl by se objevit nějaký warning do logu)